content_sync/apis/github.py

```diff
@@ -170,10 +210,11 @@ def upsert_content_files_for_user(  # pylint:disable=too-many-locals
             main_ref.edit(commit.sha)
 
             # Mark all as synced
-            for content in content_set:
-                syncstate = content.content_sync_state
-                syncstate.current_checksum = content.calculate_checksum()
-                syncstate.mark_synced()
+            for synced_state_id, filepath in synced_filepaths.items():
+                sync_state = ContentSyncState.objects.get(id=synced_state_id)
+                sync_state.current_checksum = content.calculate_checksum()
```

Comment:
I think there's a bug here, because `content` is always the content of the last item iterated from `unsynced_states`

Reply:
oops, should be `sync_state.content.calculate_checksum()`